packages/bpk-component-datepicker/src/BpkDatepicker.tsx

```diff
@@ -381,12 +380,10 @@ class BpkDatepicker extends Component<Props, State> {
             <BpkPopover
               id={`${id}-popover`}
               target={input}
-              renderTarget={renderTarget}
               onClose={this.props.onClose || this.onClose}
               isOpen={this.state.isOpen}
               label={title}
               closeButtonText={closeButtonText}
-              tabIndex={0}
```

Comment:
How come we don't need this anymore?

Reply:
This is not used by the popover anymore as the floating-ui library automatically handles it :)